vars/protecodeExecuteScan.groovy

```diff
@@ -50,14 +47,12 @@ void call(Map parameters = [:]) {
                             sh "./piper protecodeExecuteScan"
```

Comment:
In order to use piperExecuteBin we could extend piperExecuteBin that it takes a closure which replaces the default invocation of the goStep by something custom. 
```
List credentials = [
        [type: 'file', id: 'dockerCredentialsId', env: ['FILE_PATH']]
]
piperExecuteBin(parameters, STEP_NAME, METADATA_FILE, credentials){
  if (FILE_PATH) {
    File file = new File(FILE_PATH);
    def configDirPath = file.getAbsoluteFile().getParent();
    withEnv([
      "DOCKER_CONFIG=${configDirPath}",
    ]) {
      sh "./piper protecodeExecuteScan"
    }
  }
  else {
    sh "./piper protecodeExecuteScan"
  }
}
```